### PR TITLE
fix: unwrap and wq stats 

### PR DESCRIPTION
**Pull request by solidovic**
### Description

- fix(unwrap stats): hide allowance (if he is not needed)
- fix(wq stats): hide stats for l2

### Checklist:

- [x] Checked the changes locally.
- [ ] Created / updated analytics events.
- [ ] Created / updated the technical documentation (README.md / [docs](https://docs.lido.fi/) / etc.).
- [ ] Affects / requires changes in other services (Matomo / Sentry / CloudFlare / etc.).

### CODE REVIEW

File: features/withdrawals/request/form/transaction-info.tsx [hunk 1]
```diff
@@ -51,13 +57,17 @@ export const TransactionInfo = () => {
         data-testid="maxTxCost"
         loading={requestTxPriceLoading}
       >
-        <FormatPrice amount={requestTxPriceInUsd} />
+        {isWalletConnected && !isDappActiveAndNetworksMatched ? (
```
Comment (Jeday) [inline features/withdrawals/request/form/transaction-info.tsx:60]: Why is this condition different from one in claim/transactin-info?

**Review by DiRaiks — CHANGES_REQUESTED** [review]
I suggest changing the approach to writing code. The current number of flags and completely different approaches to their use leads to a lack of understanding of what is happening in the code and why. In the future this will become a magic box with a support problem. I cannot agree to merge in the form that it is now

**Review by Jeday — APPROVED** [review]

**Comment by Jeday** [comment]
@DiRaiks completely agree, I addressed those issues in https://github.com/lidofinance/ethereum-staking-widget/pull/510. We might be able to incorporate #513 in #510 to fix situation.

**Comment by solidovic** [comment]
resolved in https://github.com/lidofinance/ethereum-staking-widget/pull/510

**Pull request closed without merging**